src/frontend/cellprofiler/gui/workspace_view/_workspace_view.py

```diff
@@ -282,6 +282,16 @@ def add_row(self, row_class, rows, grid_sizer, can_delete):
                 else get_tertiary_outline_color() if row == 3
                 else wx.WHITE
             )
+        elif row_class == WorkspaceViewImageRow:
```

Comment:
Just FMI, why add a new thing rather than just change the order? Fine either way just curious